src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="layout">
   <div class="left-panel">
-    <div class="image"><img alt={{tenantName}} src="{{logo}}">
+    <div class="image"><img alt={{tenantName}} src="{{secondaryLogo}}">
```

Comment:
@sohailamjad12 
- you cannot replace `logo` with `secondaryLogo` directly
- please add conditional switching
CC: @mangeshmane @rajeevsathish

Reply:
@rajeshkumaravel  
for our case only we need two different variable name because we need two different logo in registration page  and we have already configured so, based on configuration we will get different values in two different variable ..but for other case logo variable value and secondary logo value it will be always same..it will reflect in our project only..you can check check the condition  i have given in the tenantHelper.js file